### ISSUE TITLE
Investigate missing reports with large hotspot logs

### ISSUE THREAD

**Issue by chriswhocodes** [AdoptOpenJDK/jitwatch#205]
A user has reported no results found in
- Compile chain 
- Code cache chart
- Suggestions
when large hotspot logs are processed. Small logs work fine.

Problem log can't be provided due to client confidentiality. Will try to reproduce with large log file from Eclipse.
Suspect hotspot log corruption (pre JDK9 VM logging is not thread safe and TraceClassLoading/LogCompilation/PrintAssembly can become mangled)

**Comment by danielshaya** [comment]
It turns out that the reason for the missing information was that the hotspot log was not combined into the compilation log at VM shutdown.  So we had an incomplete compilation log, missing all the <task> events and subsequently all the phase information. This had the effects noted by Chris in the issue.

On further investigation it transpires that when you stop a program with SIGQUIT (kill -9) the VM does not combine the logs.  The resolution was to kill the program with a SIGTERM (kill). 

So in summary the issue was nothing to do with the size of the logs but the way in which the process was terminated.

**Comment by chriswhocodes** [comment]
Thanks Daniel. Added this to https://github.com/AdoptOpenJDK/jitwatch/wiki/Troubleshooting
